com.sap.cloud.lm.sl.cf.core/src/test/java/com/sap/cloud/lm/sl/cf/core/validators/parameters/ApplicationNameValidatorTest.java

```diff
@@ -17,48 +20,66 @@
 
     private static final String NAMESPACE = "foo";
     private static final String APPLICATION_NAME = "bar";
+    private static final Map<String, Object> CONTEXT_APPLY_NAMESPACE = new HashMap() {
+        {
+            put(SupportedParameters.APPLY_NAMESPACE, new Boolean(true));
```

Comment:
For single key-value pair, you can use asMap method from MapUtil class in com.sap.cloud.lm.sl.common.util